code/modules/jobs/job_types/roguetown/inquisition/confessor.dm

```diff
@@ -61,13 +64,12 @@ Please do not confuse the two.
 		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/maces, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 3, TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/whipsflails, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/lockpicking, 2, TRUE)
 		H.change_stat("intelligence", -1)
-		H.change_stat("endurance", 1)
-		H.change_stat("strength", 2)
-		H.change_stat("speed", 2)
```

Comment:
I actually highly prefer this statblock change. Confessors being less speedy allows chances for heretics to flee (most towner roles don't get +2 speed), while keeping them very tanky and hard to get out of wrestling grip from.